### PR TITLE
Getallorders

### PR DESCRIPTION
**Pull request by Ivankivu**
What this PR does:- Edits an order in the list. Follow this link to see it action https://fastfood-fast-api-heroku.herokuapp.com/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,13 +8,11 @@ install:
   - pip install -r requirements.txt
```
Comment (jeanjoe) [inline .travis.yml:8]: I think u can have all the installations for your modules in the requirements.txt like `coveralls`, `codecov`, `nose` with their specific versions and `pip install -r requirements.txt` can install all of them at once.

**Review by jeanjoe — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (Ivankivu) [reply .travis.yml:8]: > I think u can have all the installations for your modules in the requirements.txt like `coveralls`, `codecov`, `nose` with their specific versions and `pip install -r requirements.txt` can install all of them at once.

Oh, that's sound great let me try that. Thank for the enlightenment

**Review by Kasulejoseph — COMMENTED** [review]
Good work but try not use static data

**Comment by Ivankivu** [comment]
@Kasulejoseph Oh ya! was only using it for development purposes thanks for the reminder

**Pull request merged**